cookiecutter/utils.py

```diff
@@ -107,7 +107,23 @@ def prompt_and_delete(path, no_input=False):
 
 
 def simple_filter(filter_function):
-    """Decorate a function to wrap it in a simplified jinja2 extension."""
+    """
+    Register a function for use as a filter extension for Jinja templates.
+
+    This decorator allows creation of a filter extension to Jinja as a function
+    instead of writing a full class extending jinja2's Extension. The function
+    name becomes the filter name
+
+    Example:
+
+    @simple_test
+    def mirror(value):
+
+    Which can then be used in jinja template as
+
+    {{ cookiecutter.param | mirror }}
```

Comment:
This needs to be a codeblock so that it renders in the API documentation. I don't think pygments has a Jinja lexer specifically, so I'm suggesting `text` here to avoid incorrect syntax highlighting:

```suggestion
    .. codeblock:: text

        {{ cookiecutter.param | mirror }}
```

This suggestion applies to the code changes lower in this file, too.